src/components/Auth/InputField/InputField.tsx

```diff
@@ -0,0 +1,87 @@
+// "use client";
+import { ChangeEvent, useState } from "react";
+import { EyeOffIcon, EyeOnIcon } from "@/src/asset";
+import Input from "./Input/Input";
+import styles from "./InputField.module.scss";
+
+interface InputField {
```

Comment:
이렇게 되면 좋을 것 같아요~
```suggestion
interface InputFieldProps {
```

현재는 타입과 컴포넌트명이 같아도 문제가 없는 것은 타입스크립트에서는 `값 영역`과 `타입 영역`이 별도로 존재하기 때문인데요~ 헷갈리는 경우가 많고 가끔 잘못 참조되는 경우가 있어서 동일한 이름으로 작성되는건 지양하면 좋을 것 같아요!